ldk/javascript/examples/self-test-loop/package.json

```diff
@@ -12,6 +12,13 @@
   "author": "",
   "ldk": {
     "permissions": {
+      "browser": {
+        "urlDomains": [
+          {
+            "value": "www.oliveai.dev"
```

Comment:
Can this be a local? Like you can open some local files or directory in a browser, just wondering if it also works here.

Reply:
Good question! I'll check on that